theurgy.lic

```diff
@@ -6,45 +6,120 @@ $settings = get_settings
 
 custom_require.call(%w(common common-items common-money common-travel drinfomon))
 
+######################## GET/PUT ########################
+
 def put_in_container(item)
-  fput "put #{item} in my #{$settings.theurgy_supply_container}"
+  DRC.bput "put #{item} in my #{$settings.theurgy_supply_container}", 'You put', 'What were you referring to'
 end
 
 def get_from_container(item)
-  fput "get #{item} from my #{$settings.theurgy_supply_container}"
+  DRC.bput "get #{item} from my #{$settings.theurgy_supply_container}", 'You get', 'I could not find', 'What were you referring to'
 end
 
-def comm_eluned
-  return unless DRStats.circle >= 3
-  return unless DRSkill.getrank('Outdoorsmanship') >= 20
+######################## COMMUNES ########################
 
+def comm_eluned
   fput "stow #{GameObj.right_hand.noun}" if GameObj.right_hand.name != 'Empty'
   fput "stow #{GameObj.left_hand.noun}" if GameObj.left_hand.name != 'Empty'
 
   DRC.forage? 'dirt'
 
   get_from_container($settings.water_holder)
-  fput 'commune eluned'
+  if 'completed this commune too recently' == DRC.bput('commune eluned', 'completed this commune too recently', 'You grind some dirt in your fist')
+    Flags['theurgy-eluned'] = true
```

Comment:
Hmm.  I did it this way because if I added the line completed this commune too recently to the flag it would trigger on all commune flags (since they share the same line).  I couldn't find any messaging that was unique between them for failed attempts.  I can try to look into a better way of doing this.